src/Verify.MSTest.SourceGenerator/TrackingNames.cs

```diff
@@ -0,0 +1,17 @@
+namespace VerifyMSTest.SourceGenerator;
+
+public static class TrackingNames
+{
+    public static string InitialTransform => nameof(InitialTransform);
+    public static string RemoveNulls => nameof(RemoveNulls);
+    public static string Collect => nameof(Collect);
+
+    public static IReadOnlyCollection<string> GetTrackingNames() =>
+        typeof(TrackingNames)
```

Comment:
wouldnt this be simpler as a hard coded array of 3 strings?

Reply:
Yes. I was trying to avoid the case where someone added a new name was added but forgot to update the array. However, that's probably not worth it. Updated to a simple array.